### PR TITLE
[TEVA-2389] new copy to clip board functionality

### PR DESCRIPTION
**Pull request by alexbowen**
https://dfedigital.atlassian.net/browse/TEVA-2389

- removes clipboard npm module and replaces with smaller script
- copy to clipboard tested in Chrome, IE, Safari, Firefox, Edge
- remove social share buttons

### CODE REVIEW

File: app/views/publishers/vacancies/_show.html.slim [hunk 1]
```diff
@@ -5,11 +5,11 @@
     .share-url-info
       .govuk-grid-row
         .govuk-grid-column-one-half
-          .share-url.display-none class="govuk-!-margin-bottom-6"
-            h2.govuk-heading-m class="govuk-!-margin-top-0" | Share this job
+          .govuk-inset-text
+            h2.govuk-heading-m class="govuk-!-margin-top-0" = "Share this job"
```
Comment (david-mears-dfe) [inline app/views/publishers/vacancies/_show.html.slim:9]: `t("jobs.share_this_job")`

Comment (david-mears-dfe) [inline app/views/publishers/vacancies/_show.html.slim:9]:
```suggestion
            h2.govuk-heading-m class="govuk-!-margin-top-0" = t("jobs.share_this_job")
```

File: app/views/publishers/vacancies/_show.html.slim [hunk 2]
```diff
@@ -5,11 +5,11 @@
     .share-url-info
       .govuk-grid-row
         .govuk-grid-column-one-half
-          .share-url.display-none class="govuk-!-margin-bottom-6"
-            h2.govuk-heading-m class="govuk-!-margin-top-0" | Share this job
+          .govuk-inset-text
```
Comment (david-mears-dfe) [inline app/views/publishers/vacancies/_show.html.slim:8]: Actually I think this whole section should be deleted?

https://ukgovernmentdfe.slack.com/archives/CG9A8H1HP/p1626087365227700?thread_ts=1626086493.226200&cid=CG9A8H1HP

**Review by david-mears-dfe — COMMENTED** [review]
Could you extract the new, shared html of app/views/publishers/vacancies/summary.html.slim and app/views/shared/vacancy/_jobseeker_view.html.slim into either a partial or a component?

File: app/views/publishers/vacancies/_show.html.slim (hunk 2 above)
Comment (alexbowen) [reply app/views/publishers/vacancies/_show.html.slim:8]: absolutely right, i didnt follow up on this convo with the actual code. does say in the ticket requirements also

**Review by csutter — APPROVED** [review]

**Pull request merged**